### ISSUE TITLE
Add banner to calculator, with 2019 and 2020 presets

### ISSUE THREAD

**Issue by cawarren** [Recidiviz/supervision-success-component#108]
_Note: This is an actual feature, and is lower priority than all of the other feature requests currently filed. Please address those before starting on this one._

**Is your feature request related to a problem? Please describe.**
Currently it's a little bit confusing to a user what this chart is about, and what they should do with it. Since it's now more specifically about pre-pandemic vs. post-pandemic levels of incarceration, we should provide some guidance to the user on how to use it.

**Describe the solution you'd like**
- Add a permanent banner at the top of the widget
- The banner should two states (dynamic components that depend on the stated selected are [bracketed]):

**If we have 2020 data for the currently selected state:**
Copy: This model estimates the change in prison population over time if supervision revocations and new admissions return to [2019] levels. To estimate the outcome of maintaining [2020] levels, set revocations to **[-21%]** and new admissions to **[-14%]**.

[Button: Reset]   [Button: Maintain 2020 levels]

**If we do not have 2020 data for the currently selected states:**
Copy: This model estimates the change in prison population if supervision revocations and new admissions return to [2019] levels. Use the sliders to see the effect of changing the parole and probation revocations or new admissions levels moving forward.

[Button: Reset]

**Button behavior**
[Maintain 2020 levels]  ← disabled if already at 2020 slider levels, otherwise clicking it sets the sliders to those levels
[Reset] ← disabled if both sliders set to 0%, otherwise clicking it sets the sliders both to 0%

**Describe alternatives you've considered**
We considered just the message, which might be a reasonable fallback if the interactive elements are too much work. The interactive elements make the tool a bit more engaging, though.

**Additional context**
Mock-up:
![image](https://user-images.githubusercontent.com/59526/131351004-60abbd02-6ca5-4d16-bb7f-a61087af1c65.png)

![image](https://user-images.githubusercontent.com/59526/131351064-a0c0d52f-d851-4de8-9390-ad557f87c500.png)

Button states:
![image](https://user-images.githubusercontent.com/59526/131351113-1e86baa8-ee2e-4ca0-bef6-dd9685756956.png)

Figma file:
https://www.figma.com/file/1HIPtEvZPS7tY1c23E58VG/Supervision-Success?node-id=508%3A158

**Comment by cawarren** [comment]
Mobile mock available here:
https://www.figma.com/file/1HIPtEvZPS7tY1c23E58VG/Supervision-Success?node-id=542%3A2
